pubsub/pulsar/metadata.go

```diff
@@ -35,6 +35,7 @@ type pulsarMetadata struct {
 	PublicKey               string                    `mapstructure:"publicKey"`
 	PrivateKey              string                    `mapstructure:"privateKey"`
 	Keys                    string                    `mapstructure:"keys"`
+	Concurrency             uint                      `mapstructure:"concurrency"`
```

Comment:
Suggest calling this `maxConcurrentHandlers` which is the same name used in Azure Service Bus PubSub (and note that the default value there is 0, meaning unlimited)

Reply:
Renamed

Reply:
Thanks for the rename.

Yup, sounds good to keep 100 for now. As discussed on Discord, can you please open an issue for 1.14 to allow this to be "0" (i.e. unlimited) too? Right now it's using a buffered channel which was probably not the best implementation.